### PR TITLE
DrivAerML: 4L/256d+T_max=30 — lr=5.5e-4 (fine-grained LR search)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

lr=5e-4 is currently the best DrivAerML learning rate, and lr=6e-4 is being tested elsewhere. The gap between them is narrow — lr=5.5e-4 sits exactly in between. DrivAerML's LR sensitivity is well-established (Lion at lr=3e-4 degraded epoch-over-epoch while AdamW at lr=5e-4 converges), so even a 10% change in LR could meaningfully affect final quality over 45 epochs. This micro-tuning experiment fills the gap systematically.

## Instructions

Run the standard 4L/256d DrivAerML config with `--lr 5.5e-4` (only change from baseline):

```bash
cd target/icml2026 && python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 5.5e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group drivaerml-4L256d-fine-lr
```

Report `val_primary/surface_rel_l2_pct` at every epoch. Note the epoch at best val metric.

## Baseline

- **Primary metric:** `val_primary/surface_rel_l2_pct` (lower is better)
- **Current best:** 12.70% (val) / 13.54% (test) — PR #2593
- **Configuration:** 4L/256d, T_max=30, AdamW **lr=5e-4**, no-EMA, Fourier, 45 epochs
- **Baseline W&B run:** 3aaevlho
- **Reproduce baseline:**
  ```
  cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --model-layers 4 --model-hidden-dim 256 --model-heads 4 --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200
  ```

## CRITICAL OOM Flags

Always include these or you will OOM:
```
--batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200 --model-heads 4
```

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2659 because ADVISOR: Closing — DrivAerML baseline shifted to 5.73% (4L/384d, PR #2602). All 4L/256d experiments are now obsolete — the best 4L/256d result was 11.97%. Width scaling is the dominant lever. Redirecting your GPU to 4L/384d experiments. You'll receive a new assignment shortly..
